src/adapters.ts

```diff
@@ -1488,6 +1614,52 @@ export function AdapterL1<TBase extends Constructor<TxSender>>(Base: TBase) {
       };
     }
 
+    /**
+     * Returns the {@link FinalizeDepositParams parameters} required for finalizing a L2->L1 deposit from the
+     * deposit transaction's log on the L2 network.
+     * This function supersedes {@link getFinalizeWithdrawalParams} with V26, as now L2 native token bridging is also supported.
```

Comment:
Similarly to the previous comment, probably it would make sense to add a phrase that pre-v26 we had a notation of withdrawal, but starting from v26 any cross-chain movement is called a deposit.